tests/components/glances/test_sensor.py

```diff
@@ -43,6 +43,8 @@ async def test_sensor_states(hass: HomeAssistant) -> None:
         assert state.state == HA_SENSOR_DATA["raid"]["md1"]["available"]
     if state := hass.states.get("sensor.0_0_0_0_md1_used"):
         assert state.state == HA_SENSOR_DATA["raid"]["md1"]["used"]
+    if state := hass.states.get("sensor.0_0_0_0_uptime"):
```

Comment:
Please remove the if statement as you expect the state to have the same value as `A_SENSOR_DATA["uptime"]`.

Currently, the state is None as an exception is raised.

Reply:
> Should the unit test not fail when one of these values is None and thereby does not match the expected value?

We expect the test to fail, but the if statement prevents it from failing in this case.

Reply:
It is expected not to fail, but to match the given data... I understand that the if clause can prevent tests from failing when the data is None. I mean that there is no point for the other sensors preventing it from failing (by an exception). A cleaner solution would be to use `assert state := hass.states.get("sensor.0_0_0_0_uptime") is not None` before checking that the value matches the expected value.